scripts/defaultScripts.js

```diff
@@ -7,6 +7,7 @@
 //
 //  Copyright 2014 High Fidelity, Inc.
 //  Copyright 2020 Vircadia contributors.
+//  Copyright 2022 Overte contributors.
```

Comment:
```suggestion
//  Copyright 2022 Overte e.V.
```
I would suggest already using the organization name, since it it will be able to actually enforce the license, while "Overte contributors" isn't an actual entity.

Reply:
Changes done 
(on 3 other files too)